drdt-vagrant/README.md

```diff
@@ -0,0 +1,98 @@
+This is the DRDT Vagrant Box.
+
+Will serve as a base box for the DRDT project.
+
+all environnement will be based off this box.
+
+	Ubuntu 18.04
+	apache2
+	php 7.1
+	MySQL
+
+
+Requirements/How to Install :
+__________________
+
+	What will be installed :
+
+	Fonctional LAMP Server. 
+		-apache2
+		-Ubuntu 18.04
+		-php 7.1
+		-MySQL
+
+	Full Wordpres installation of drdt.constructionprotips.com
+
+	Redis cache server.
+
+	1 ) Add 192.168.205.11 drdt.constructionprotips.com in your /etc/hosts file.
+
+	2 ) Latest version of Vagrant is required in order to setup the private network with ubuntu 18.04.
+
+	3 ) Latest version of VirtualBox
+
+	4 ) The vagrant diskresize plugin is required.
+
+		To install :
+
+		vagrant plugin install vagrant-diskresize
+
+	5 ) You will need a valid copy of the CPT database you can download one here :
+
+		http://staging.constructionprotips.com/wp-content/uploads/sites/9/2018/12/CPT_dump.sql
+
+		*Currently some issue downloading the file, I'm working on finding a good place to store this. Will probably be in one of our s3 storage on AWS.
+
+		you will need to place that file in the drdt-vagrant/ folder
+	
+	6 ) You will need to clone the drdt-test project inside the drdt-vagrant directory.
```

Comment:
The VM is in the same repo as the code. Mind clarifying?

Reply:
This is the way I have always seen a vagrant config, dvlpr-vagrant works the same.
I'll look into getting this to run one level above!